vscode-extension/README.md

```diff
@@ -70,6 +70,13 @@ Path to a Relay config relative to the `rootDirectory`. Without this, the compil
 
 An array of project configuration in the form `{name: string, rootDirectory: string, pathToConfig: string}`. If omitted, it is assumed your workspace uses a single Relay config and the compiler will search for your config file. But you can also use this configuration if your Relay config is in a nested directory. This configuration must be used if your workspace has multiple Relay projects, each with their own config file.
 
+#### `relay.pathToExtraDataProviderScript` (default: `null`)
+
+Path to a script to look up the actual definition for a GraphQL entity for implementation-first GraphQL schemas. This script will be called for "goto definition" requests to the LSP instead of opening the schema.
+The script will be called with 2 arguments. The first will be the relay project name, the second will be either "Type" or "Type.field" (a type or the field of a type, repectively).
```

Comment:
Maybe we should future-proof this by calling with some command like "lookup default Foo.bar" instead of hardcoding this as a lookup action? I say this because I see there's a "stats" thing in the ExtraDataProvider trait which might warrant this sort of future proofing

Reply:
I'd be fine making this a "locate" command and exposing a separate option for a command to support the query info.

Reply:
Should this option clarify that you need a new-ish version of relay-compiler for this to work?